scripts/preview-generator.ts

```diff
@@ -38,31 +40,45 @@ const renderPreviewBySlug = async (slug: string, override = false) => {
     return
   }
 
-  if (question.choiceLeft.img_path.indexOf('webp') > -1 || question.choiceRight.img_path.indexOf('webp') > -1) {
+  if (question.choiceLeft.img_path.endsWith('.webp') || question.choiceRight.img_path.endsWith('.webp')) {
```

Comment:
I did not know the `endsWith` method ! Nice !!

Reply:
Happy to teach you something! 😁